pkg/workflow/tasks/discover.go

```diff
@@ -143,3 +162,27 @@ func NewViewTaskDiscover(pd *packages.PackageDiscover, cli client.Client, cfg *r
 		templateLoader:     templateLoader,
 	}
 }
+
+// GetSuspendStepDelayDuration get suspend step delay duration
+func GetSuspendStepDelayDuration(step v1beta1.WorkflowStep) (bool, builtintime.Duration, error) {
```

Comment:
I think it's might be better if we add a `getTaskParameter` function in `taskRunner`, then we can use tr.getTaskParameter to get duration.

Reply:
As we communicated earlier, this change can not meet all requirements, i will ignore it.